app/views/devise/sessions/new.html.erb

```diff
@@ -11,39 +11,18 @@
 </div>
  
 <div class="header_image">
-  
 </div>  
 
 <div class="log_in_box">
-  
-  <div class="box_tab">
-    LOG IN
-  </div>
-
-  <div class="form_box">
-    <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-      <div class="">
-        <%= f.email_field :email, autofocus: true, placeholder: "Username or Email" %>
-      </div>
-
-      <div >
-        <%= f.password_field :password, autocomplete: "off", placeholder: "Password"  %>
-      </div>
-
-      <div class="form_buttons">
-        <div class="remember_button">
-        <% if devise_mapping.rememberable? -%>
-            <%= f.check_box :remember_me %>
-            <%= f.label :remember_me %>
-        <% end -%>
-        </div>
-        <div class="form_buttons">
-          <%= f.submit("Log in", class: 'selectButton') %>
-        </div>
-      </div>
-    <% end %>
-  </div>
-
-  <%= render "devise/shared/links" %>
-  
+  <%- if devise_mapping.omniauthable? %>
```

Comment:
are these dashes (`-`) in these erb tags serving a purpose? if not, can we remove them?

Reply:
Looks like they are not needed, interesting discussion on all the variations though:
https://stackoverflow.com/questions/7996695/what-is-the-difference-between-and-in-erb-in-rails